ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -343,4 +384,21 @@ - (NSString * _Nullable)clientID{
     
     return (_clientID) ? _clientID : nil;
 }
+
+#pragma mark - Helpers
+- (void)setExpirationDateWith:(NSNumber*)seconds{
+    self.keychainData.expirationDate = [NSDate dateWithTimeIntervalSinceNow:[seconds doubleValue]];
+}
+
+- (UIViewController*) topMostController
```

Comment:
Additionally, I'm not sure if we should use this here, it is related to the getAccessToken feature